src/test/java/cn/edu/tsinghua/tsfile/timeseries/write/ReadPageInMemTest.java

```diff
@@ -227,6 +227,12 @@ public void MultiDeltaObjectTest() {
 		assertEquals(null, innerWriter.query("root.car.d1", "s5").get(1));
 		assertEquals(null, innerWriter.query("root.car.d2", "s5").get(0));
 		assertEquals(null, innerWriter.query("root.car.d2", "s5").get(1));
+		try {
```

Comment:
It's needless here. Messages of thrown exception will be shown and this test case won't be passed, even if you don't catch this exception explicitly.

Reply:
modified